src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Logging/RazorLogHubLoggerProvider.cs

```diff
@@ -25,38 +26,43 @@ internal sealed class RazorLogHubLoggerProvider : IRazorLoggerProvider
 
     private readonly RazorLogHubTraceProvider _traceProvider;
     private readonly RazorLogger _razorLogger;
-    private readonly AsyncQueue<(TraceEventType Level, string Message, object[] Args)> _outputQueue;
+    private readonly Queue<(TraceEventType Level, string Message, object[] Args)> _messageQueue;
     private readonly CancellationTokenSource _disposalTokenSource;
+    private TraceSource? _traceSource;
 
     [ImportingConstructor]
     public RazorLogHubLoggerProvider(RazorLogHubTraceProvider traceProvider, RazorLogger razorLogger)
     {
         _traceProvider = traceProvider ?? throw new ArgumentNullException(nameof(traceProvider));
         _razorLogger = razorLogger ?? throw new ArgumentNullException(nameof(razorLogger));
 
-        _outputQueue = new();
-        _disposalTokenSource = new CancellationTokenSource();
+        _messageQueue = new();
```

Comment:
what was broken with the last PR that this commit is now fixing?

Reply:
Using an async queue was a mistake, because the trace message was written on a different context than it was originally logged. In the most recent commit there are no more queues.